src/python/src/pyclient.cpp

```diff
@@ -94,12 +100,12 @@ py::array PyClient::get_tensor(const std::string& key)
         tensor = _client->_get_tensorbase_obj(key);
     }
     catch(const std::exception& e) {
-        throw std::runtime_error(e.what());
+        throw smart_runtime_error(e.what());
```

Comment:
We seem to repeat this code stucture for handing errors alot... is there any way to reduce the amount of boilerplate here? template? wrapper?

Reply:
Alas, no